src/pages/index.js

```diff
@@ -0,0 +1,123 @@
+import Head from "next/head";
+import Modal from "@/components/Modal";
+import ProfilePic from "../../public/assets/profile-pic.jpg";
+import Skills from "@/components/Skills";
+import Projects from "@/components/Projects";
+import Footer from "@/components/Footer";
+import NavBar from "@/components/NavBar";
+import ScrollToTop from "react-scroll-to-top";
+
+//react icons
+import { GiMoonBats } from "react-icons/gi";
+import {
+  AiFillLinkedin,
+  AiFillGithub,
+  AiFillCodeSandboxCircle,
+} from "react-icons/ai";
+import { FaFreeCodeCamp } from "react-icons/fa";
+import { FaArrowUp } from "react-icons/fa";
+
+import React, { useState } from "react";
+
+//to include an img in next you have to import the next/image component
+import Image from "next/image";
+import Testimonials from "@/components/Testimonials";
+
+export default function Home() {
+  const [openModal, setOpenModal] = useState(false);
+
+  //dark theme
+  const [darkMode, setDarkMode] = useState(false);
+
+  return (
+    <div className={darkMode ? "dark" : ""}>
+      <Head>
+        <title>Octavian Busuioc Portofolio</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <NavBar />
+      <main className="bg-white px-10 dark:bg-gray-900">
+        <section className="min-h-screen">
+          <nav className="py-10 mb-12 flex justify-between px-32">
+            <div className="flex gap-2"></div>
```

Comment:
În JSX, spre deosebire de HTML, poți pune self-closing tag la orice element. Mie îmi arată mai bine decât conținut gol.

```suggestion
            <div className="flex gap-2" />
```